packages/template-retail-react-app/app/ssr.js

```diff
@@ -54,7 +54,7 @@ const options = {
     // Set this to false if using a SLAS public client
     // When setting this to true, make sure to also set the PWA_KIT_SLAS_CLIENT_SECRET
     // environment variable as this endpoint will return HTTP 501 if it is not set
-    useSLASPrivateClient: true,
+    useSLASPrivateClient: false,
```

Comment:
The current client ID in the config is a public client, do we want to change this back to `false`?